_pages/dev/ssr/server-side-rendering-contextual-logging.md

```diff
@@ -236,17 +221,19 @@ import { ErrorHandlingModule } from '@spartacus/setup/ssr';
 export class SpartacusModule {}
 ```
 
-By importing `ErrorHandlingModule`, `CxErrorHandler` will handle all errors that occur during Server Side Rendering, and these errors will be logged with an appropriate context.
+By importing `ErrorHandlingModule`, the `CxErrorHandler` then handles all errors that occur during server-side rendering, and these errors are logged with an appropriate context.
+
+**Note:** Since user applications may contain their own implementations of `ErrorHandler`, the `CxErrorHandler` is not used by default when contextual logging is enabled. To use the `CxErrorHandler`, it is recommended that you extend your error handler to use the `LoggerService`.
```

Comment:
I changed a bit the meaning of this sentence.
`To use CxErrorHandler, it is recommended that you extend your handler to use the LoggerService`
Actually, we want to tell the users if they want to have context attached to errors, they should use `LoggerService` i their custom implementation of error handlers.

An example:
```
**Note:** Since user applications may contain their own implementations of `ErrorHandler`, the `CxErrorHandler` is not used by default when contextual logging is enabled. To enhance errors with the context, it is recommended that you extend your error handler to use the `LoggerService`.
```

Reply:
Done.